time_tracker_api/time_entries/time_entries_repository.py

```diff
@@ -290,28 +287,31 @@ def find_running(
             "owner_id": owner_id,
             "tenant_id": tenant_id,
         }
+
+        query_builder = (
+            TimeEntryQueryBuilder()
+            .add_sql_non_existent_attribute_condition('end_date')
```

Comment:
yes!. This condition seems to make sense only  in TimeEntries context. Some business context : we consider timeentries with no end-date or end-date=null to be `running-time-entries` the condition could be `add_sql_is_running_time_entry_condition()` with no parameters and be added in TimeEntriesQueryBuilder. :)

Reply:
Resolved with the changes agreed at the meeting.